[+] Add tensorflow-cc port (Linux only)

nartes opened the pull request:
https://github.com/Microsoft/vcpkg/issues/3048

Commit:
[+] Add tensorflow-cc port (Linux only)

msftclas commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/not_signed)](https://cla.opensource.microsoft.com/Microsoft/vcpkg?pullRequest=5222) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/Microsoft/vcpkg?pullRequest=5222) before we can accept your contribution.<br/><br/>:x: nartes [sign now](https://cla.opensource.microsoft.com/Microsoft/vcpkg?pullRequest=5222)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/Microsoft/vcpkg?pullRequest=5222) it.</sub>

ras0219-msft committed:
[tensorflow] Work in progress

Commit:
[~] Refactor installation of tensorflow/external

  1. Install tensorflow/external to tensorflow-etc/external;
  2. Fix TensorflowCCConfig.cmake accordingly.

ras0219-msft committed:
Merge remote-tracking branch 'origin/master' into dev/roschuma/5222

jasjuang commented:
thanks for the effort, is there any chance that this can work on windows as well?

dan-shaw commented:
See #7568
